session_items.py

```diff
@@ -1,9 +1,24 @@
-from flask import session
+from flask import current_app
```

Comment:
File name doesn't seem terribly relevant anymore as this is for the trello api and not the flask cookie session state.